<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="sempr::gui::RawComponentWidget" name="tabRawComponent">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
     </widget>
     <widget class="sempr::gui::TriplePropertyMapWidget" name="tabTriplePropertyMap">
      <attribute name="title">
       <string>Page</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>sempr::gui::RawComponentWidget</class>
   <extends>QWidget</extends>
   <header>../src/RawComponentWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>sempr::gui::TriplePropertyMapWidget</class>
   <extends>QWidget</extends>
   <header>../src/TriplePropertyMapWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
